arm64 support?

Hi

I am having troubling with arm64 support. I would like to use image optimization, but I get an error when I'm enabling this. It looks like it is disabled by default? Which is the opposite of what Ghost documentation states it is.

Image: ghost:4 (tested for alpine and debian)
`"Image": "sha256:b07775c88fc91a454477ce2947a9a7242d77e1f0140f6ffc79e601c2cfff35fb"`

The error message that I receive is that `sharp is not installed`. Why isn't sharp installed in the default image for arm64?

```
InternalServerError: Sharp wasn't installed
    at Object.resizeFromPath (/var/lib/ghost/versions/4.11.0/node_modules/@tryghost/image-transform/lib/transform.js:81:31)
    at normalize (/var/lib/ghost/versions/4.11.0/core/server/web/api/middleware/normalize-image.js:24:20)
    at Layer.handle [as handle_request] (/var/lib/ghost/versions/4.11.0/node_modules/express/lib/router/layer.js:95:5)
    at next (/var/lib/ghost/versions/4.11.0/node_modules/express/lib/router/route.js:137:13)
    at uploadValidation (/var/lib/ghost/versions/4.11.0/core/server/web/api/middleware/upload.js:88:9)
    at Layer.handle [as handle_request] (/var/lib/ghost/versions/4.11.0/node_modules/express/lib/router/layer.js:95:5)
    at next (/var/lib/ghost/versions/4.11.0/node_modules/express/lib/router/route.js:137:13)
    at /var/lib/ghost/versions/4.11.0/core/server/web/api/middleware/upload.js:42:9
    at Immediate._onImmediate (/var/lib/ghost/versions/4.11.0/node_modules/multer/lib/make-middleware.js:53:37)
    at processImmediate (internal/timers.js:466:21)

Error: Cannot find module 'sharp'
Require stack:
- /var/lib/ghost/versions/4.11.0/node_modules/@tryghost/image-transform/lib/transform.js
- /var/lib/ghost/versions/4.11.0/node_modules/@tryghost/image-transform/index.js
- /var/lib/ghost/versions/4.11.0/core/server/lib/mobiledoc.js
- /var/lib/ghost/versions/4.11.0/core/server/models/post.js
- /var/lib/ghost/versions/4.11.0/core/server/models/index.js
- /var/lib/ghost/versions/4.11.0/core/boot.js
- /var/lib/ghost/versions/4.11.0/ghost.js
- /var/lib/ghost/versions/4.11.0/index.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:889:15)
    at Function.Module._load (internal/modules/cjs/loader.js:745:27)
    at Module.require (internal/modules/cjs/loader.js:961:19)
    at require (internal/modules/cjs/helpers.js:92:18)
    at Object.resizeFromPath (/var/lib/ghost/versions/4.11.0/node_modules/@tryghost/image-transform/lib/transform.js:79:9)
    at normalize (/var/lib/ghost/versions/4.11.0/core/server/web/api/middleware/normalize-image.js:24:20)
    at Layer.handle [as handle_request] (/var/lib/ghost/versions/4.11.0/node_modules/express/lib/router/layer.js:95:5)
    at next (/var/lib/ghost/versions/4.11.0/node_modules/express/lib/router/route.js:137:13)
    at uploadValidation (/var/lib/ghost/versions/4.11.0/core/server/web/api/middleware/upload.js:88:9)
    at Layer.handle [as handle_request] (/var/lib/ghost/versions/4.11.0/node_modules/express/lib/router/layer.js:95:5)
    at next (/var/lib/ghost/versions/4.11.0/node_modules/express/lib/router/route.js:137:13)
    at /var/lib/ghost/versions/4.11.0/core/server/web/api/middleware/upload.js:42:9
    at Immediate._onImmediate (/var/lib/ghost/versions/4.11.0/node_modules/multer/lib/make-middleware.js:53:37)
    at processImmediate (internal/timers.js:466:21)
```

--- a/4/alpine/Dockerfile
+++ b/4/alpine/Dockerfile
@@ -51,9 +51,9 @@ RUN set -eux; \
 	sqlite3Version="$(node -p 'require("./package.json").optionalDependencies.sqlite3')"; \
 	if ! su-exec node yarn add "sqlite3@$sqlite3Version" --force; then \
 # must be some non-amd64 architecture pre-built binaries aren't published for, so let's install some build deps and do-it-all-over-again
-		apk add --no-cache --virtual .build-deps g++ gcc libc-dev make python3 vips-dev; \
+		apk add --no-cache --virtual .build-deps g++ gcc libc-dev make python2 vips-dev; \
 		\
-		npm_config_python='python3' su-exec node yarn add "sqlite3@$sqlite3Version" --force --build-from-source; \
+		npm_config_python='python2' su-exec node yarn add "sqlite3@$sqlite3Version" --force --build-from-source; \
 		\
 		apk del --no-network .build-deps; \
 	fi; \

--- a/4/debian/Dockerfile
+++ b/4/debian/Dockerfile
@@ -1,7 +1,7 @@
 # https://docs.ghost.org/faq/node-versions/
 # https://github.com/nodejs/Release (looking for "LTS")
 # https://github.com/TryGhost/Ghost/blob/v4.1.2/package.json#L38
-FROM node:14-buster-slim
+FROM node:14-bullseye-slim
 
 # grab gosu for easy step-down from root
 # https://github.com/tianon/gosu/releases
@@ -77,10 +77,10 @@ RUN set -eux; \
 # must be some non-amd64 architecture pre-built binaries aren't published for, so let's install some build deps and do-it-all-over-again
 		savedAptMark="$(apt-mark showmanual)"; \
 		apt-get update; \
-		apt-get install -y --no-install-recommends g++ gcc libc-dev libvips-dev make python3; \
+		apt-get install -y --no-install-recommends g++ gcc libc-dev libvips-dev make python2; \
 		rm -rf /var/lib/apt/lists/*; \
 		\
-		npm_config_python='python3' gosu node yarn add "sqlite3@$sqlite3Version" --force --build-from-source --ignore-optional; \
+		npm_config_python='python2' gosu node yarn add "sqlite3@$sqlite3Version" --force --build-from-source --ignore-optional; \
 		\
 		apt-mark showmanual | xargs apt-mark auto > /dev/null; \
 		[ -z "$savedAptMark" ] || apt-mark manual $savedAptMark; \

